frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -932,6 +932,22 @@ describe("scenarios > question > filter", () => {
     cy.findByRole("button", { name: "Done" }).should("not.be.disabled");
   });
 
+  it.skip("custom expression filter should work with numeric value before an operator (metabase#15893)", () => {
+    cy.intercept("POST", "/api/dataset").as("dataset");
+
+    openOrdersTable({ mode: "notebook" });
+    cy.findByText("Filter").click();
+    cy.findByText("Custom Expression").click();
+    cy.get("[contenteditable=true]")
+      .type("0 < [ID]")
+      .blur();
+    cy.findByRole("button", { name: "Done" }).click();
```

Comment:
I don't think it should allow me to click "Done", since this is a MBQL limitation, so we want to block users from continuing and probably display a nice error, but this will probably work too.

Reply:
If we end up fixing this correctly, the test will fail on this step so it should still be a good repro :)

Reply:
Obviously, it would block the merge.
But that should be a good clue to the dev who ends up fixing this issue that something's off and that this test should be updated.

I'm never sure how to approach repros for features we don't currenly have or for things like this where we **currently** have a certain behavior that should/will change.

Reply:
I'm pretty sure that MBQL won't be changed to address this - it would just be the CE that would block such usage.